test/e2e/atlas/serverless_test.go

```diff
@@ -105,12 +105,12 @@ func TestServerless(t *testing.T) {
 		resp, err := cmd.CombinedOutput()
 		req.NoError(err, string(resp))
 
-		var clusters mongodbatlas.ClustersResponse
+		var clusters []atlasv2.ServerlessInstanceDescription
```

Comment:
this is a change of behavior and a bad one

Reply:
We would need more details? Do you meant that object is not generic enough?

Reply:
for example this is running this branch 
```bash
./bin/atlas serverless ls -P dev
Error: template: output:1:29: executing "output" at <.Results>: can't evaluate field Results in type []mongodbatlasv2.ServerlessInstanceDescription
```

Reply:
This is the template used, and I missed changing the `.Results`:
```
var listTemplate = `ID	NAME	MDB VER	STATE{{range .Results}}
{{.Id}}	{{.Name}}	{{.MongoDBVersion}}	{{.StateName}}{{end}}
```

So it looks like we can safely unwrap the envelope